samples/04_gis_analysts_data_scientists/detecting-deforestation-using-kmeans-clustering-on-sentinel-imagery.ipynb

```diff
@@ -0,0 +1,1169 @@
+{
```

Comment:
<p>The raster should exist on pythonapi playground portal</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/975/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
raster added to playground portal